MobileBuy/sample/src/xApollo/java/com/shopify/sample/SampleApplication.java

```diff
@@ -34,6 +36,16 @@ public void onCreate() {
     super.onCreate();
   }
 
+  @Override
+  protected UseCases createUseCases() {
+    throw new RuntimeException("Not implement yet");
```

Comment:
I think this must be implemented in this PR as well?

Reply:
If you think we should refactor two build flavors at the same time, then it should.

Reply:
Yes and no, if we merge it then `develop` branch is kinda in broken state

Reply:
True. That is why I would like to have dedicate branch for this ticket before merge it to develop. But let me see. I can change both flavors at the same time. It is a good way to test the structure with both Shopify SDK and Apollo.